app/models/event.rb

```diff
@@ -106,4 +106,7 @@ def hide_cover_fees?
 		nonprofit.hide_cover_fees? || misc_event_info&.hide_cover_fees_option
 	end
 
+	def custom_get_tickets_button_label
+		misc_event_info&.custom_get_tickets_button_label
+	end
```

Comment:
I think it'd make more sense for the option for the label to be defined here.

SO this would be somethign like:`

```ruby
def get_tickets_button_label
   misc_event_info&.custom_get_tickets_button_label || 'Get tickets'
end
```

Generally, it's easier to have the logic in the model because because it's really hard to test erb templates in detail.